### ISSUE TITLE
Issue with Color Overlap in IGV Genome Track

### ISSUE THREAD

**Issue by yzhirui** [igvteam/igv#1602]
Hi, 
I am using IGV version 2.18.2 on linux. When plotting reads per BAM file, I noticed that the genome track exhibits color overlap, as shown in the attached picture.

![IGV_question](https://github.com/user-attachments/assets/4a86000b-361b-4f9e-8356-6530b1ec5bb7)

I also checked the initial bases of this region:
AGTCCAGCTGTGGACAGTGCTTCAGAGAGG
It's not caused by degenerate bases.

Could this overlap be due to alternative alleles in relative positions? How can I change this display to show the original genome bases instead?

And my bat file like this：
```
new
preference SAM.FILTER_DUPLICATES false
genome genome.fa
snapshotDirectory IGV_plot
load a.bam
load b.bam
load c.bam
snapshot plot.png
exit
```

Thank you!

**Comment by jrobinso** [comment]
Hi, sorry but I really don't know what you are showing us.  What is that a picture of?

**Comment by yzhirui** [comment]
> Hi, sorry but I really don't know what you are showing us. What is that a picture of?

Hi! Thanks for your quick response! The image is a snapshot of the genome sequence. I've included a complete example below.
![test](https://github.com/user-attachments/assets/4c2b0fe3-cd85-455b-9540-7e29aa63c538)

 Is that clearer now?

**Comment by jrobinso** [comment]
Yes that is clear.  You are not zoomed in far enough to see individual bases.

**Comment by yzhirui** [comment]
> Yes that is clear. I don't know your screen resolution but you are possibly not zoomed in far enough to see individual bases.

Ok, you are totally right. I used 300bp before, and when zoomed in to 100bp, this problem has been resolved.
Thank you again for your kindly help！
